cmd/nerdctl/run.go

```diff
@@ -511,6 +515,12 @@ func runAction(clicontext *cli.Context) error {
 		}
 	}
 
```

Comment:
I think it is better to check the len of `strutil.DedupeStrSlice(clicontext.StringSlice("ulimit"))` here

Reply:
Why?

Reply:
simply, no need to call `generateUlimitsOpts` if len == 0